### PR TITLE
Feature/3 col tables

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-71.dev.c2s.nhschoices.net/gp-surgeries/41235](http://profiles-pr-71.dev.c2s.nhschoices.net/gp-surgeries/41235)

File: .dockerignore [hunk 1]
```diff
@@ -6,4 +6,4 @@ npm-debug.log.*
 .DS_Store
 *.swp
 .idea/*
-
+lib/mobileDetect.js
```
Comment (st3v3nhunt) [inline .dockerignore:9]: Does this file really not want to be deployed into the container?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node app.js",
     "lint": "eslint --ext .js,.json .",
-    "test": "NODE_ENV=test mocha --recursive test",
+    "test": "snyk test && NODE_ENV=test mocha --recursive test",
```
Comment (st3v3nhunt) [inline package.json:9]: Is there a specific reason for adding `snyk test` to the `test` script? I would prefer it to not be done there as it is a relatively expensive operation and one that we are moving towards doing on `prepush` only

File: package.json [hunk 3]
```diff
@@ -71,4 +72,4 @@
     "nodemon": "^1.11.0"
   },
   "snyk": true
-}
+}
```
Comment (st3v3nhunt) [inline package.json:75]: New line please

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
The usage of `detect-mobile-browser` for an express app looks more simple than what has been done here. Checkout https://www.npmjs.com/package/detect-mobile-browser

The file `app/lib`mobileDetect.js` isn't required. The `requires` can be updated as per the examples. I'd get rid of the test too as that is implicit in the use of the library.

File: package.json (hunk 2 above)
Comment (ajthomascouk) [reply package.json:9]: Not sure I put it there.

File: .dockerignore (hunk 1 above)
Comment (ajthomascouk) [reply .dockerignore:9]: Probably put there by mistake.

**Comment by st3v3nhunt** [comment]
@ajthomascouk Can you rebase and finish the changes please?

**Comment by ajthomascouk** [comment]
@st3v3nhunt wrote:
> @ajthomascouk Can you rebase and finish the changes please?

Yup, had higher priorities.

**Comment by ajthomascouk** [comment]
Make days bold on mobile/stacked view.

**Comment by ajthomascouk** [comment]
Closing this PR and starting fresh.

**Pull request closed without merging**